### PR TITLE
Serial term prompt move

### PR DESCRIPTION
**Pull request by richiejp**
This should be merged shortly after https://github.com/os-autoinst/os-autoinst/pull/797

@jtzhao @pevik

### CODE REVIEW

File: lib/serial_terminal.pm [hunk 1]
```diff
@@ -43,8 +29,8 @@ escape sequences (i.e. a single #) and changes the terminal width.
 
 =cut
 sub login {
-    die 'Login expects one argument' unless @_ == 1;
-    my $user = shift || 'root';
+    die 'Login expects two argument' unless @_ == 2;
```
Comment (pevik) [inline lib/serial_terminal.pm]: typo: argument => arguments

**Comment by pevik** [comment]
LGTM.

**Comment by coolo** [comment]
only merge after deployment

**Comment by asmorodskyi** [comment]
@coolo  mentioned PR was merged 9 days ago , can you confirm that it was already deployed ?

**Comment by coolo** [comment]
yeah, we should be fine

**Pull request merged**